### ISSUE TITLE
Is it possible to train the pixelCNN with this code base, rather than using the supplied download url? 

### ISSUE THREAD

**Issue by andreasrobinson** [kimiyoung/ssl_bad_gan#2]
Or alternatively is there another opensource code base you'd recommend for training the pixelCNN?

**Comment by kimiyoung** [comment]
Yes! We have a training script for pixelCNN, and we will release it asap. Thanks for pointing this out.

**Comment by andreasrobinson** [comment]
Thanks!

**Comment by yg320** [comment]
Hi,

Do you have any idea when this script will be released?

Thanks! This code looks great :)

**Comment by 331801070049** [comment]
Have you managed to run the mnist_trainer.py program?
